content/references/rippled-api/api-conventions/error-formatting.md

```diff
@@ -101,15 +101,16 @@ For other errors that returned with HTTP status code 200 OK, the responses are f
 
 All methods can potentially return any of the following values for the `error` code:
 
-* `amendmentBlocked` - The server is [amendment blocked](amendments.html#amendment-blocked) and needs to be updated to the latest version to stay synced with the XRP Ledger network.
+- `amendmentBlocked` - The server is [amendment blocked](amendments.html#amendment-blocked) and needs to be updated to the latest version to stay synced with the XRP Ledger network.
+- `failedToForward` - ([Reporting Mode][] servers only) The server tried to forward this request to a P2P Mode server, but the connection failed.
 - `invalid_API_version` - The server does not support the [API version number](request-formatting.html#api-versioning) from the request.
-* `jsonInvalid` - (WebSocket only) The request is not a proper JSON object.
-    * JSON-RPC returns a 400 Bad Request HTTP error in this case instead.
-* `missingCommand` - (WebSocket only) The request did not specify a `command` field.
-    * JSON-RPC returns a 400 Bad Request HTTP error in this case instead.
-* `noClosed` - The server does not have a closed ledger, typically because it has not finished starting up.
-* `noCurrent` - The server does not know what the current ledger is, due to high load, network problems, validator failures, incorrect configuration, or some other problem.
-* `noNetwork` - The server is having trouble connecting to the rest of the XRP Ledger peer-to-peer network (and is not running in stand-alone mode).
-* `tooBusy` - The server is under too much load to do this command right now. Generally not returned if you are connected as an admin.
-* `unknownCmd` - The request does not contain a [command](rippled-api.html) that the `rippled` server recognizes.
-* `wsTextRequired` - (WebSocket only) The request's [opcode](https://tools.ietf.org/html/rfc6455#section-5.2) is not text. <!-- SPELLING_IGNORE: opcode -->
+- `jsonInvalid` - (WebSocket only) The request is not a proper JSON object.
+    - JSON-RPC returns a 400 Bad Request HTTP error in this case instead.
+- `missingCommand` - (WebSocket only) The request did not specify a `command` field.
+    - JSON-RPC returns a 400 Bad Request HTTP error in this case instead.
+- `noClosed` - The server does not have a closed ledger, typically because it has not finished starting up.
```

Comment:
Not sure if the above JSON-RPC bullet entries render correctly due to indentation.

Reply:
Confirmed it renders as intended:
![2021-02-18_151834_154864167](https://user-images.githubusercontent.com/7515597/108434526-a80c2e80-71fc-11eb-8150-782c1c602a2d.png)